business_requirement/README.rst

```diff
@@ -0,0 +1,265 @@
+
+* Construction
+* Trading (New product development)
+* Business Consultancy
+* IT development
+
+What is the difference between a BR and CRM lead?
+
+* CRM leads are sales oriented
+* BR are project and workload estimation oriented
+
+How to use this module?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This module only contains the standard models for business requirement:
+
+* BR model definition
+* Standard setup and views
+* Standard Workflow
+
+
+|image0|
+
+.. |image0| image:: static/img/bus_req.png
+   :width: 800 px
+   :alt: Business Requirement Form
+
+Installation
+============
+
+No specific steps required
+
+Configuration
+=============
+
+Users
+^^^^^
+
+* **Business Requirement User**: can create and confirm a BR
+* **Business Requirement Manager**: can create, confirm and approve a BR
+
+Alias and emails
+^^^^^^^^^^^^^^^^
+You can setup an alias in Settings/Technical/Email/Alias in order to create 
+business requirements directly from email received at a specific address.
+
+|image1|
+
+.. |image1| image:: static/img/bus_req_alias.png
+   :width: 800 px
+   :alt: Email Alias setup
+
+You can start conversation with the followers in the chatter area of the 
+BR like in tasks, issue or CRM leads.
+
+Sequences
+^^^^^^^^^
+
+Default numbering format: BR123456.
+
+You can modify the default numbering sequence in Settings/Technical/Sequences&Identifier/Sequences.
+
+Search for Business Requirement sequence and alter it if necessary.
+
+Tags
+^^^^
+
+You can create and assign tags for your business requirements in Business Requirements/Configuration/Bus. Req. Category
+
+|image2|
+
+.. |image2| image:: static/img/bus_req_tags.png
+   :width: 600 px
+   :alt: Define Tags
+
+
+Master project
+^^^^^^^^^^^^^^
+
+You can define a master project linked to the business requirement.
+
+Followers from the project will automatically be added to the business requirements.
+
+
+Usage
+=====
+
+Simple BR
+^^^^^^^^^
+1. Define the header information
+
+* Master Project (Partner is the one related to the master project)
+* Priority (low, Medium, High)
+* Change request: Is it a change request? (currently only informational with n model or action)
+* Tags: any relevant tag for the business.
+* Owner and approver by default
+
+|image3|
+
+.. |image3| image:: static/img/bus_req_tags2.png
+   :width: 800 px
+   :alt: Input header information
+
+2. Input the customer story, scenario and gap (simple html editor with image and text)
+
+|image4|
+
+.. |image4| image:: static/img/bus_req_cust_story.png
+   :width: 600 px
+   :alt: Input customer story, scenario, gap
+
+3. Confirm the Business requirement (for BR User and Manager)
+   At that stage the Customer story/Scenario/Gap is not modifiable anymore
+
+|image5|
+
+.. |image5| image:: static/img/bus_req_confirmed.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+
+4. Approve the Business requirement (for BR Manager)
+
+|image6|
+
+.. |image6| image:: static/img/bus_req_approved.png
```

Comment:
Image is not rendered correctly. Should be:
.. |image6| image:: business_requirement/static/img/bus_req_approved.png

Apply for all other images in .rst